ERROR - 2024-04-28 07:39:47 --> Error connecting to the database: mysqli_sql_exception: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 07:39:47 --> Unable to connect to the database.
Main connection [MySQLi]: Host 'AUUNNG' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 07:39:54 --> Error connecting to the database: mysqli_sql_exception: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 07:39:54 --> Unable to connect to the database.
Main connection [MySQLi]: Host 'AUUNNG' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 07:41:48 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101:8...', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 07:41:48 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 07:45:39 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101:8...', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 07:45:39 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 08:06:35 --> Error connecting to the database: mysqli_sql_exception: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 08:06:35 --> Unable to connect to the database.
Main connection [MySQLi]: Host 'AUUNNG' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 08:06:35 --> Error connecting to the database: mysqli_sql_exception: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 08:06:35 --> Unable to connect to the database.
Main connection [MySQLi]: Host 'AUUNNG' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 08:06:36 --> Error connecting to the database: mysqli_sql_exception: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.1.101', 'root', '', 'classflow', 3306, '', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#6 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#7 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#11 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host 'AUUNNG' is not allowed to connect to this MariaDB server in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ClassFlow\system\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', Array, NULL, 'username')
#5 C:\xampp\htdocs\ClassFlow\system\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', Array, Array)
#6 C:\xampp\htdocs\ClassFlow\system\Controller.php(140): CodeIgniter\Validation\Validation->run()
#7 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate(Array)
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#9 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#10 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-04-28 08:06:36 --> Unable to connect to the database.
Main connection [MySQLi]: Host 'AUUNNG' is not allowed to connect to this MariaDB server
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Validation\StrictRules\Rules.php(206): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\Validation\Validation.php(278): CodeIgniter\Validation\StrictRules\Rules->is_unique('admin', 'username', [...], null, 'username')
 5 SYSTEMPATH\Validation\Validation.php(182): CodeIgniter\Validation\Validation->processRules('username', 'Username', 'admin', [...], [...])
 6 SYSTEMPATH\Controller.php(140): CodeIgniter\Validation\Validation->run()
 7 APPPATH\Controllers\TeacherController.php(105): CodeIgniter\Controller->validate([...])
 8 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
 9 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
10 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ClassFlow\\public\\index.php')
ERROR - 2024-04-28 08:09:12 --> mysqli_sql_exception: Unknown column 'created_date' in 'field list' in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\app\Models\TeacherModel.php(62): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(113): App\Models\TeacherModel->register(Array)
#6 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#7 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
ERROR - 2024-04-28 08:10:02 --> mysqli_sql_exception: Unknown column 'created_date' in 'field list' in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\app\Models\TeacherModel.php(62): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(113): App\Models\TeacherModel->register(Array)
#6 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#7 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
ERROR - 2024-04-28 08:10:49 --> mysqli_sql_exception: Unknown column 'created_date' in 'field list' in C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\ClassFlow\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ClassFlow\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ClassFlow\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ClassFlow\app\Models\TeacherModel.php(62): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ClassFlow\app\Controllers\TeacherController.php(113): App\Models\TeacherModel->register(Array)
#6 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(932): App\Controllers\TeacherController->register('$1')
#7 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TeacherController))
#8 C:\xampp\htdocs\ClassFlow\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ClassFlow\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ClassFlow\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
